app/assets/javascripts/steps.js

```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```

Comment:
If you don't need this, you can delete it. Same for all other files. Check here https://github.com/WDI-HK-8/JiujitsMi/pull/5/files